setup.py

```diff
@@ -30,6 +30,7 @@
         'pyyaml',
         'six',
         'webob',
+        'web-fragments',
```

Comment:
FYI, the PR for web fragments is: https://github.com/andy-armstrong/web-fragments/pull/1